config/example/game_options.txt

```diff
@@ -74,6 +74,6 @@ BOMBCAP 20
 ### ROUNDSTART SILICON LAWS ###
 ## This controls what the AI's laws are at the start of the round.
 ## Set to 0/commented for "off", silicons will just start with Crewsimov.
-## Set to 1 for "random", silicons will start with a random lawset picked from (at the time of writing): Nanotrasen Default, P.A.L.A.D.I.N., Corporate, Robop and Crewsimov. More can be added by changing the law datum "default" variable in ai_laws.dm.
+## Set to 1 for "random", silicons will start with a random lawset picked from (at the time of writing): Robocop and Crewsimov. More can be added by changing the law datum "default" variable in ai_laws.dm.
```

Comment:
If you're updating this comment, might as well update that it is actually `default` not `random` that required the binary flag change.